### PR TITLE
Do not translate constant functions from <0.4.0 as payable

### LINKED ISSUE #500: ABI with `constant` and `payable` equal to true
With Solidity 0.3.3, using the `update` from `import { update } from 'solc/abi'`
produces this abi (if `constant` is originally true) - see below.
Having both `constant` and `payable` to `true` throw an error while trying to use it in ethers 5.0.

```
constant: true
inputs: []
name: "get"
outputs: [{…}]
payable: true
type: "function"
```

### PR DESCRIPTION
**Pull request by axic**
Fixes #500.

@yann300 can you try this?

### CODE REVIEW

File: test/abi.js [hunk 1]
```diff
@@ -15,11 +15,16 @@ tape('ABI translator', function (t) {
...
   });
-  t.test('0.3.6 (function)', function (st) {
+  t.test('0.3.6 (non-constant function)', function (st) {
     var input = [ { inputs: [], type: 'function' } ];
     st.deepEqual(abi.update('0.3.6', input), [ { inputs: [], payable: true, stateMutability: 'payable', type: 'function' }, { payable: true, stateMutability: 'payable', type: 'fallback' } ]);
     st.end();
   });
+  t.test('0.3.6 (constant function)', function (st) {
+    var input = [ { inputs: [], type: 'function', constant: true } ];
+    st.deepEqual(abi.update('0.3.6', input), [ { inputs: [], payable: true, constant: true, stateMutability: 'payable', type: 'function' }, { payable: true, stateMutability: 'payable', type: 'fallback' } ]);
```
Comment (axic) [inline test/abi.js]: This was actually translated to `payable: true, constant: true` :)

**Comment by yann300** [comment]
that seems to be ok ;)

**Comment by yann300** [comment]
by any chance, do you plan to get a release of that soon?
cc @Aniket-Engg

**Comment by axic** [comment]
> by any chance, do you plan to get a release of that soon?

Perhaps next week we have a 0.7 release, but I guess worst case it has to be within the next two weeks.

**Review by leonardoalt — APPROVED** [review]

**Comment by axic** [comment]
@yann300 @aniket-engg released in 0.7.5

**Pull request merged**